dev/main.tf

```
@@ -83,6 +83,13 @@ module "aws-iam-identity-center" {
       family_name      = "Ma"
       email            = "mark.xianghui.ma@gmail.com"
     },
+    jasonw0030 : {
+      group_membership = ["DevOps"]
+      user_name        = "jasonw0030"
+      given_name       = "Jason"
+      family_name      = "Wang"
+      email            = "wangzj0703@gmail.com"
+    }
```

```suggestion
    },
```
Always add a comma at the end of your block, this make everyone's commit separate and clean